cmd/restic/cmd_stats.go

```diff
@@ -282,9 +292,10 @@ func verifyStatsInput(gopts GlobalOptions, args []string) error {
 // to collect information about it, as well as state needed
 // for a successful and efficient walk.
 type statsContainer struct {
-	TotalSize      uint64 `json:"total_size"`
-	TotalFileCount uint64 `json:"total_file_count"`
-	TotalBlobCount uint64 `json:"total_blob_count,omitempty"`
+	TotalSize             uint64 `json:"total_size"`
+	TotalUncompressedSize uint64 `json:"total_uncompressed_size"`
```

Comment:
Shouldn't this also be marked as `omitempty`?

Reply:
I think it should, fixed, thanks.